lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnContainerManager.java

```diff
@@ -74,6 +80,7 @@
 
   private final YarnConfiguration yarnConf;
   private final AMRMClientAsync<AMRMClient.ContainerRequest> resourceManager;
+  private final YarnProxyUser yarnProxyUser;
```

Comment:
No. Client and Driver have different injectors and thus will have different instances of the `YarnProxyUser` class. We still need to copy the credentials from one to another, hence the changes in the API.